app/src/main/java/ar/com/wolox/android/example/ui/home/HomeFragment.kt

```diff
@@ -0,0 +1,69 @@
+package ar.com.wolox.android.example.ui.home
+
+import androidx.core.util.Pair
+import ar.com.wolox.android.R
+import ar.com.wolox.android.example.ui.home.news.NewsFragment
+import ar.com.wolox.android.example.ui.home.profile.ProfileFragment
+import ar.com.wolox.wolmo.core.adapter.viewpager.SimpleFragmentPagerAdapter
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import ar.com.wolox.wolmo.core.presenter.BasePresenter
+import com.google.android.material.tabs.TabLayout
+import kotlinx.android.synthetic.main.fragment_home.*
+import javax.inject.Inject
+
+/**
+ * Home Fragment
+ */
+
+class HomeFragment @Inject constructor() : WolmoFragment<BasePresenter<Any>>() {
+
+    @Inject internal lateinit var page1news: NewsFragment
+    @Inject internal lateinit var page2profile: ProfileFragment
+    private lateinit var fragmentHomePagerAdapter: SimpleFragmentPagerAdapter
+
+    override fun layout(): Int {
+        return R.layout.fragment_home
+    }
+
+    override fun init() {
+        vToolbarFragmentHome.setLogo(R.drawable.wolox_logo)
+        vToolbarFragmentHome.setTitle(R.string.example_wolox)
+
+        vTabLayoutFragmentHome.setupWithViewPager(vViewPagerFragmentHome)
+
+        fragmentHomePagerAdapter = SimpleFragmentPagerAdapter(childFragmentManager)
```

Comment:
With Kotlin you can make the initialization before the assignment with the `Kotlin standard functions`. [Here](https://medium.com/@elye.project/mastering-kotlin-standard-functions-run-with-let-also-and-apply-9cd334b0ef84) is a link with more information about that. Let use it. You could use an `apply` for this.